### PR TITLE
Make tooltip a usable component

### PR DESCRIPTION
**Pull request by nathf**

### CODE REVIEW

File: src/components/ux-tooltip/use-cases/dataDriven.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
 		"tagElement": "",
```
Comment (scott-dickson) [inline src/components/ux-tooltip/use-cases/dataDriven.json:5]: What is this used for?  I can't see it referenced in the .hbs?

File: src/components/ux-tooltip/use-cases/dataDriven.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
...
 		"className": "tooltip",
 		"selectorName": "tooltip_demo",
-		"content": "This is a <strong>tooltip content</strong>.",
+		"content": "This is a <b>tooltip content</b>.",
```
Comment (ivanwills) [inline src/components/ux-tooltip/use-cases/dataDriven.json]: Why bold over string?

**Comment by ivanwills** [comment]
This mostly looks done any reason for not finishing?

File: src/components/ux-tooltip/use-cases/dataDriven.json (hunk 1 above)
Comment (ivanwills) [reply src/components/ux-tooltip/use-cases/dataDriven.json:5]: tagElement is used by the tooltip decorator

**Pull request merged**